lib/mmseqs/src/commons/CSProfile.cpp

```diff
@@ -87,7 +87,7 @@ void ContextLibrary::readContextProfile(std::stringstream &in, LibraryReader &re
     size_t nalph = reader.ReadInt(line.c_str(), "ALPH", "Unable to parse CRF state 'ALPH'!");
     if (nalph != 20){
         Debug(Debug::ERROR) << "Alphabet size of serialized CRF state should be 20 "
-                "but is acutally" << nalph << "!\n";
+                "but is actually" << nalph << "!\n";
```

Comment:
Please submit the lib/mmseqs fixes to MMseqs2.

Reply:
MR open.